pkg/services/ngalert/remote/alertmanager.go

```diff
@@ -94,12 +100,15 @@ func NewAlertmanager(cfg AlertmanagerConfig, orgID int64) (*Alertmanager, error)
 		return nil, err
 	}
 
+	// We won't be handling files on disk, we can pass an empty string as workingDirPath.
+	stateStore := notifier.NewFileStore(orgID, kvstore, "")
```

Comment:
nit: Here it is! I believe we agreed that it's better for the `remoteAlertmanager` to accept the filestore directly given it has a smaller interfaces and we don't really use the kvstore.